internal/integration/mysql_test.go

```diff
@@ -26,6 +28,7 @@ type myTest struct {
 	db      *sql.DB
 	drv     *mysql.Driver
 	version string
+	port    int
```

Comment:
Store and use the dsn instead?

Reply:
I like the port,
a. it can be used for something else,
b. it requires "calculating" the string and all the other params are just "passed" not "calculated"